go/libkb/wot.go

```diff
@@ -198,48 +212,35 @@ func fetchWot(mctx MetaContext, username *string) (res []serverWotVouch, err err
 	return response.Vouches, nil
 }
 
-func FetchMyWot(mctx MetaContext) (res []keybase1.WotVouch, err error) {
-	defer mctx.Trace("FetchMyWot", func() error { return err })()
-	serverVouches, err := fetchWot(mctx, nil)
+// FetchUserWot gets vouches written for vouchee (if specified) by voucher (if
+// specified).
+func FetchUserWot(mctx MetaContext, vouchee *string, voucher *string) (res []keybase1.WotVouch, err error) {
```

Comment:
also, i suggest we call this entry point something more like `FetchWotVouches`. it's a little more general, implying that the arguments can drive more different behavior.

Reply:
and if we want convenience behavior, it's just as clear to define those as special args (or functions that generate the correct args -- see `NewLoadUserSelfArg`) without the added risk of someone coming along in the future and making ever more "convenience" methods.

Reply:
looking at `LoadUserArg` and `NewLoadUserSelfArg`, there are many more parameters there than we have for fetching vouches right now that i'm reluctant to add those layers of abstraction in this PR. i could just get rid of this convenience method (which is only used by the react method)

Reply:
sure. in the case of loading a user, it's a no brainer to abstract the parameters behind a typed arg. here, it's a little more about predicting the future, and doing the things that are (1) least surprising, and (2) make follow-on work easier. my instincts are that now is the time to build this particular level of structure/abstraction: far enough along that we know the interface (basically what's coming in and basically what's going out), and early enough that it can provide real value to future work.

Reply:
i added `libkb.FetchWotVouchesArg`